tests/pytest.ini

```diff
@@ -0,0 +1,11 @@
+[pytest]
+testpaths = .
+addopts =
+    --cov=pycompwa
+    --cov-config=.coveragerc
+    --durations=0
+    --cov-report=html
+markers =
+    slow: marks tests as slow (deselect with '-m "not slow"')
```

Comment:
So slow tests are also run by default and `-m "not slow"` turns them off?

Reply:
Yes, this is a [custom marker](https://docs.pytest.org/en/latest/mark.html#registering-marks), it is not selected by default.

Reply:
Yeah ok I just gave it a try